src/app/productConfig/productConfig.modal.js

```diff
@@ -19,7 +19,7 @@ class ModalInstanceCtrl{
   }
 
   frequencyOrder(f){
-    let order = ['NA', 'MON', 'QUARTERLY', 'ANNUAL', 'SEMIANNUAL'];
+    let order = ['NA', 'MON', 'QUARTERLY', 'SEMIANNUAL', 'ANNUAL'];
```

Comment:
Should `SEMIANNUAL` still be here? I just don't know the details of what you're doing.